linux_os/guide/services/docker/docker_selinux_enabled/rule.yml

```
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: rhel7,rhel8
+prodtype: rhel7,rhel8,rhel9
```

AFAIK Docker is only in RHEL 7. Moreover the rule has a RHEL7 - only OVAL. I suggest restricting this prodtype to rhel7.
```suggestion
prodtype: rhel7
```